index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
```

Comment:
[OPTIONAL] Kindly delete 8px or margin around the body, this value come as default and it create an overflow like below.
![Screen Recording 2022-12-02 at 2 05 25 PM](https://user-images.githubusercontent.com/25887253/205279432-f90c69ce-41c2-490d-b584-41c579e74ba6.gif)